pyth-sdk/src/lib.rs

```diff
@@ -177,4 +197,20 @@ impl PriceFeed {
             expo:  self.expo,
         }
     }
+
+    /// Get the "unchecked" previous aggregate price with Trading status.
+    ///
+    /// Returns the previous aggregate price with the timestamp it was generated. The price might
+    /// be invalid or inaccurate at the current time; You need to check timestamp when using it.
+    /// Please use `get_current_price` where possible.
```

Comment:
I think a slightly stronger warning is safer here:

```
    /// Get the most recent "unchecked" previous price with Trading status,
    /// along with the timestamp at which it was generated.
    ///
    /// WARNING: you should ONLY consider using this in situations where the
    /// current price is not TRADING, so `get_current_price` will return None.
    /// We make no guarantees about the unchecked price and confidence returned by
    /// this function: it could differ significantly from the true price.
    /// We strongly encourage you to use `get_current_price` instead, in the vast
    /// majority of situations.
```

Reply:
I read it like this: 
use this in situations where the current price is not TRADING, so `get_current_price` will return None.

:D

Reply:
Ah ok, maybe leave out the first sentence of the warning then 😄 

```
    /// Get the most recent "unchecked" previous price with Trading status,
    /// along with the timestamp at which it was generated.
    ///
    /// WARNING:
    /// We make no guarantees about the unchecked price and confidence returned by
    /// this function: it could differ significantly from the true price.
    /// We strongly encourage you to use `get_current_price` instead, in the vast
    /// majority of situations.
```

Reply:
Wonderful. Thanks.